tasks.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
I'd remove this whole file, an example with the additional properties in the readme file should be enough. Since we can't really assume what's on the users' system.

Reply:
i just wanted to show what the config file would look like with the options i added.
guess i could have just edited the readme.